.github/workflows/android_builds.yml

```diff
@@ -0,0 +1,201 @@
+          tar -xjf x86_64-godot-linux-gnu_sdk-buildroot.tar.bz2
+          cd x86_64-godot-linux-gnu_sdk-buildroot
+          sed -i x86_64-godot-linux-gnu/sysroot/usr/lib/pkgconfig/dbus-1.pc -e "s@/lib@/lib64@g"
+          ./relocate-sdk.sh
+          cd ..
+
+      # [Already provided by the custom buildroot]
+      # Azure repositories are not reliable, we need to prevent azure giving us packages.
+      # - name: Make apt sources.list use the default Ubuntu repositories
+      #   run: |
+      #     sudo rm -f /etc/apt/sources.list.d/*
+      #     sudo cp -f misc/ci/sources.list /etc/apt/sources.list
+      #     sudo apt-get update
+
+      # [Already provided by the custom buildroot]
+      # Install all packages (except scons)
+      # - name: Configure dependencies
+      #   run: |
+      #     sudo apt-get install build-essential pkg-config libx11-dev libxcursor-dev \
+      #       libxinerama-dev libgl1-mesa-dev libglu-dev libasound2-dev libpulse-dev libudev-dev libxi-dev libxrandr-dev yasm
+
+      # Upload cache on completion and check it out now
+      - name: Load .scons_cache directory
+        id: android-template
+        uses: actions/cache@v2
+        with:
+          path: ${{github.workspace}}/.scons_cache/
+          key: ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}-${{github.sha}}-${{matrix.target}}-${{matrix.arch}}
+          restore-keys: |
+            ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}-${{github.sha}}-${{matrix.target}}-${{matrix.arch}}
+            ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}
+            ${{github.job}}-${{env.GODOT_BASE_BRANCH}}
+
+      # [Already provided by the custom buildroot]
+      # Use python 3.x release (works cross platform; best to keep self contained in it's own step)
+      # - name: Set up Python 3.x
+      #   uses: actions/setup-python@v2
+      #   with:
+      #     # Semantic version range syntax or exact version of a Python version
+      #     python-version: '3.x'
+      #     # Optional - x64 or x86 architecture, defaults to x64
+      #     architecture: 'x64'
+
+      # [Already provided by the custom buildroot]
+      # Setup scons, print python version and scons version info, so if anything is broken it won't run the build.
+      # - name: Configuring Python packages
+      #   run: |
+      #     python -c "import sys; print(sys.version)"
+      #     python -m pip install scons
+      #     python --version
+      #     scons --version
+
+      # We should always be explicit with our flags usage here since it's gonna be sure to always set those flags
+      - name: Compilation
+        env:
+          SCONS_CACHE: ${{github.workspace}}/.scons_cache/
+        run: |
+          PATH=${GITHUB_WORKSPACE}/x86_64-godot-linux-gnu_sdk-buildroot/bin:$PATH
+          scons -j2 verbose=yes warnings=extra platform=android target=${{matrix.target}} arch=${{matrix.arch}}
```

Comment:
The arch names used in Godot's build system are `"x86_32", "x86_64", "arm32", "arm64"` with some aliases to help users pick the correct name. You are currently using `"x86", "x86_64", "armv7", "arm64v8"` in this pull request. Ideally, the names the CI uses should be the same as what Godot uses, since these are the names the final executables will have. So please change this.